<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Patient</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="selectButton">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Patient Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="patientName"/>
      </item>
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout_10">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Additional Notes:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QPlainTextEdit" name="patientNotes"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
